convert-config/entrypoint.sh

```diff
@@ -65,8 +65,28 @@ objects:
   done
 done
 
+# generate schemas
+# Clone ksl-schema-language repo
+git clone --depth=1 https://github.com/project-kessel/ksl-schema-language.git
+
+# Build KSL compiler binary
+cd ksl-schema-language
+mkdir -p bin/
+go build -gcflags "all=-N -l" -o ./bin/ ./...
+cd ..
+
+# Run KSL compiler
+
+./ksl-schema-language/bin/ksl -o ${INPUT_KSL_SRC_STAGE}/schema.zed ${INPUT_KSL_SRC_STAGE}/src/*.ksl ${INPUT_KSL_SRC_STAGE}/src/*.json
```

Comment:
How about getting rid of this ksl_src_stage and do something similar to iterate through stage and prod and get the path:
PERMISSION_DIR=${WORKSPACE}/configs/${ENV}/permissions
May make the code more consistent

Reply:
Makes sense, easier to follow with less variables between actions to keep track of.